roles/common/tasks/main.yaml

```diff
@@ -19,13 +19,17 @@
     - name: Parse tfstate
       set_fact: 
         tfstate: "{{ lookup('tfstate', TFSTATE) }}"
+    - name: default LOADBALANCER_SOURCE_RANGES
+      set_fact:
+        LOADBALANCER_SOURCE_RANGES: ["0.0.0.0/0"]
```

Comment:
Is there a way to have a default that's more secure? Wondering about the effects of having the default being wide open.

Reply:
Not really. If the user does not provide LOADBALANCER_SOURCE_RANGES then the deployment would not be reachable at all. So, we default to 0.0.0.0/0. This is essentially what the same as what the tools do when no loadBalancerSourceRanges are set.

Reply:
Thanks for the info. That's what I needed to know.